### ISSUE TITLE
Remote Extension is not working in Insider build

### ISSUE THREAD

**Issue by dhairyanadapara** [microsoft/vscode#92423]
- Version: 1.44.0-insider (user setup)
- Commit: b0791e5236f462fbb785e9a00319705607cefb9d
- Date: 2020-03-10T11:23:10.498Z
- Electron: 7.1.11
- Chrome: 78.0.3904.130
- Node.js: 12.8.1
- V8: 7.8.279.23-electron.0
- OS: Windows_NT x64 10.0.18363

Steps to Reproduce:

1. Open VSCode
2. Open Remote Extension
3. No hosts are shown in list

Error in Remote Extension. 
```
ERR command 'opensshremotes.openEmptyWindow' not found: Error: command 'opensshremotes.openEmptyWindow' not found
    at e._tryExecuteCommand (file:///C:/Users/dhairya.nadapara/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:3559:697)
    at file:///C:/Users/dhairya.nadapara/AppData/Local/Programs/Microsoft VS Code Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:3559:578
```

![image](https://user-images.githubusercontent.com/19250981/76383000-79e16600-6380-11ea-95fb-5fb76ee1fa13.png)

**Comment by sanket-bhalerao** [comment]
i am getting following error.
i could connect to the server before update to 1.44 insiders build.
![image](https://user-images.githubusercontent.com/9264813/76385271-58d04380-6387-11ea-92de-35f4192bcc1e.png)

**Comment by sanket-bhalerao** [comment]
it is caused by extn Remote-ssh
downgrade it to earlier version and remote works fine.
![image](https://user-images.githubusercontent.com/9264813/76388412-190d5a00-638f-11ea-8ac5-0491061ec2c1.png)

**Comment by dhairyanadapara** [comment]
@sanket-bhalerao Tried downgrading the `Remote-SSH` extension to `0.49.0` and `0.48.0`. But, still it's not working.

**Comment by sanket-bhalerao** [comment]
it seems to work for me, that is the only extn i have downgraded. for me the version is 0.49.0 (i downgraded from 0.50.0)

**Comment by sanket-bhalerao** [comment]
Have you reloaded the window once the downgrade is completed?

**Comment by dhairyanadapara** [comment]
I have checked that but that's also not working. I looked more and found that it is coming due to antivirus. It shows that `extensionHostProcess.js` file in `C:\Users\<username>\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\workbench\services\extensions\node` have virus with name `JS.Nemucod.AF`. Due to this antivirus is adding the file into `Quarantine`

**Comment by roblourens** [comment]
Thanks for figuring that out. Which antivirus are you using? It's a false alarm.

**Comment by dhairyanadapara** [comment]
We are using `Seqrite` (https://www.seqrite.com/)

**Comment by roblourens** [comment]
Ok, hopefully you are able to whitelist that file/folder, this needs to be fixed on Seqrite's end.

**Comment by vsbmef** [comment]
![Screenshot_2020-04-11-20-30-45-008_com android chrome](https://user-images.githubusercontent.com/52532137/79047841-dca37700-7c36-11ea-8e35-cdac61d547a8.jpg)

Today when i started using vs code it was too. Giving same error. So I check for it. And I found that its having some file as virus. Im using Quick Heal Anti virus  . Virus Name - JS.Nemucod.AF
When ever i use to update my vs code. It gives same error. So I installed downgrade version of Vs code. And it is working fine.
